### PR TITLE
Added check mechanism for missing CO Person ID variable in debug mode

### PR DESCRIPTION
**Pull request by ioigoume**
In debug mode the co person id is not available through the request but through the session. As a result i added an if statement to fetch the value i need per situation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,3 +52,4 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - When multiple idps are included in the request data, as a unified string with a semicolon delimiter, we do not parse them properly in order to retrieve only the last idp
 - The update of the AuthnAuthority attribute gets overwritten by the old value
 - Attribute string length to 256 characters, so as to much the max size of an Entity Id
+- CoGrous Search functionality broke in debug mode due to missing variable
```
Comment (NicolasLiampotis) [inline CHANGELOG.md]: Fix CoGroups Search functionality in debug mode due to missing variable

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
LGTM but please fix typo in CHANGELOG

File: CHANGELOG.md (hunk 1 above)
Comment (ioigoume) [reply CHANGELOG.md]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**